torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -436,6 +437,22 @@ std::shared_ptr<SugaredValue> toSugaredValue(
   }
 }
 
+std::shared_ptr<SugaredValue> CUDAPythonModuleValue::attr(
+    const SourceRange& loc,
+    Function& m,
+    const std::string& field) {
+
+  if(field != "getCurrentStream") {
+    py::object member = getattr(loc, field);
+    return toSugaredValue(member, m, loc, /*is_constant=*/true);
+  }
+  // note: is_constant = true because we consider that global properties
+  // on modules like math.pi or torch.float to be constants
+  // even though it is possible, though rare, for someone to mutate them
+  auto cur_stream = make_custom_class<torch::jit::CUDAStream>(0);
```

Comment:
This creates the stream statically at compile time and ignores the arguments passed in by the user. Instead of creating the stream during compilation, I think what you should do is insert a call to the custom operator that you added into the graph and return a `std::shared_ptr<SugaredValue>` containing its output. That will dynamically create the stream with the desired device and priority.